FastTimeSeries/src/models/InceptionTime.jl

```diff
@@ -0,0 +1,83 @@
+"""
+    InceptionModule(ni::Int, nf::Int, ks::Int = 40, bottleneck::Bool = true)
+
+TBW
+"""
+function InceptionModule(ni::Int, nf::Int, ks::Int = 40, bottleneck::Bool = true)
+    ks = [ks ÷ (2^i) for i in range(0, stop = 2)]
+    ks = [ks[i] % 2 == 0 ? ks[i] - 1 : ks[i] for i in range(1, stop = 3)]  # ensure odd ks
```

Comment:
```suggestion
    ks = [iseven(ks[i]) ? ks[i] - 1 : ks[i] for i in 1:3]  # ensure odd ks
```
Almost always prefer range syntax to the function (I haven't added suggestions for above and below, but this applies there too). I think it may also help not to use the same variable name (`ks`) for the original input parameter and your subsequent list of kernel sizes.